apps/storybook/src/TiledHeatmapMesh.stories.tsx

```diff
@@ -131,6 +131,53 @@ class MandelbrotTilesApi extends TilesApi {
   }
 }
 
+class CheckboardTilesApi extends TilesApi {
```

Comment:
For early tests, I used tiles showing a checkboard pattern so I kept it as a story.

I seem to remember that you had something similar at one point so don't hesitate to replace/correct it.

Reply:
Yes, I did the same at first!
At that time, I didn't bother to use a store, I don't think it is needed.

Reply:
Fine, I removed the store :+1: